client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationIndicatorsColumn/SelfInvestigationIcon/SelfInvestigationIcon.tsx

```diff
@@ -18,7 +18,7 @@ const SelfInvestigationIcon = (props: Props) => {
 	const [anchorEl, setAnchorEl] = React.useState<HTMLButtonElement | null>(null);
 
 	const classes = useStyles();
-	const handleTooltipClose = (e : React.MouseEvent<Document, MouseEvent>) => {
+	const handleTooltipClose = (e: any) => {
```

Comment:
why any?

Reply:
TS giving me errors because onClick requires one type and onClose another and they don't fit to each other... so when I'm trying to give more specific types I get typescript errors